Sming/SmingCore/FileSystem.cpp

```diff
@@ -115,16 +115,22 @@ void fileClearLastError(file_t fd)
 	SPIFFS_clearerr(&_filesystemStorageHandle);
 }
 
-void fileSetContent(const String& fileName, const String& content)
+signed int fileSetContent(const String& fileName, const String& content)
```

Comment:
@kmihaylov Just use `int` instead of `signed int`. In C/C++ `int` represents signed integers. Apply that also to the other changes coming from you.